docs/acp/user/intro-to-acp/developing-an-app.md

```diff
@@ -0,0 +1,142 @@
+```
+The first line ensures that you are in the correct working directory. The second line initializes the project, and the third line runs the workflow.
+
+Starting from SprykerSDK version 0.3.0, you can use the following set:
+
+```bash
+cd /www/my-app (The local project directory you created before)
+spryker-sdk sdk:workflow:run # and select PBC workflow
+```
+The workflow guides you as much as possible through the process of building an app. Whenever something needs manual interaction, the workflow stops with a message on what you need to do. After you completed the manual step, re-run the workflow with the `spryker-sdk sdk:workflow:run` command. This continues the previously paused workflow.
+
+### Workflow details
+
+The `spryker-sdk sdk:workflow:run` command guides you through the whole process of building an app. This command does the following.
+
+#### 1. Downloads the boilerplate code and adds app definition files
+
+One of the first steps of the workflow is downloading a reduced Spryker project into the working directory. It already contains a lot of code that lets you start implementing your business logic.
+This step can take a few minutes to complete.
+
+{% info_block infoBox "Info" %}
+
+At this point, no dependencies are installed. 
+
+{% endinfo_block %}
+
+There are a couple of files that an app must have. The workflow guides you through creating them.
+
+**Manifest files**
+The manifest files define the details like title, category, and description of the app itself. The manifest files are also used to display information about the app on the App Catalog Page and the App Detail Page in the Back office.
+
+For more details on the manifest files, see [App manifest files](LINK)
+
+{% info_block infoBox "Info" %}
+
+You need to update the manifest file manually. The SDK only creates a boilerplate file.
+
+{% endinfo_block %}
+
+**Translation file**
+The translation file contains the keys and the translation values for each locale in which the app should be displayed. 
+
+This file is created with the help of a wizard, but you can also add or update it manually.
+
+For more details, see [App Configuration Translation](LINK)
+
+**Configuration file**
+The configuration file defines the form that is displayed in the Back Office App Catalog after the App was connected and needed some configuration. 
+
+This file is created with the help of a wizard, but you can also add or update it manually.
+
+For more details, see [App Configuration](LINK)
+
+![acp-sdk-workflow-1](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/dev/developing-an-app/ACP-SDK-Workflow-black-1.jpg)
+
+#### 2. Creates app API
+
+The command defines SyncAPI and AsyncAPI.
+
+**SyncAPI**
+The SyncAPI defines the app's synchronous endpoints, also known as Glue endpoints. The workflow creates a boilerplate that you need to update with the required endpoints your PBC should have. See the [current OpenAPI specification](https://spec.openapis.org/oas/v3.1.0).
```

Comment:
"synchronous endpoints, also known as Glue endpoints" it's only Spryker terminology, so maybe "synchronous endpoints,  or Glue endpoints (in Spryker terminology)"